app/admin/regional_holiday.rb

```diff
@@ -36,6 +43,7 @@
       f.input :day
       f.input :month
       f.input :offices, as: :check_boxes, collection: Office.active.order(:city)
+      f.input :cities, as: :check_boxes, collection: City.all.order(:name)
```

Comment:
this might list a lot of cities, maybe is better to use a select box

Reply:
Done!

![image](https://user-images.githubusercontent.com/40892570/233441337-85fd2bf1-85d1-4380-9b13-975c9d4329dd.png)